content/en/blog/_posts/2022-05-13-grpc-probes-in-beta.md

```diff
@@ -48,7 +48,7 @@ The natively supported health probe has many benefits over the workaround involv
 
 With the native gRPC support you don't need to download and carry `10MB` of an additional executable with your image.
 Exec probes are generally slower than a gRPC call as they require instantiating a new process to run an executable.
-It also makes the checks less sensible for edge cases when the pod is running at maximum resources and has troubles
+It also makes the checks less sensible for edge cases when the pod is running with minimum resources and has troubles
```

Comment:
Why should we make this change? It looks wrong.

Reply:
This is reverted. Sorry, I had a wrong understanding about comments from @divya-mohan0209 above.

Reply:
@sftim & @windsonsea I was confused by the original statement, which is why I suggested the change. I assumed edge cases = cases of Kubernetes installs on the Edge instead of extreme cases. That's on me, my bad.